x-pack/plugins/index_management/common/index.ts

```diff
@@ -5,6 +5,9 @@
  * 2.0.
  */
 
+// TODO: https://github.com/elastic/kibana/issues/110892
+/* eslint-disable @kbn/eslint/no_export_all */
+
```

Comment:
Same here

Reply:
Again, very bottom of the file, `export * from './types';`